bin/console

```diff
@@ -7,7 +7,7 @@ require "ingram_micro"
 # with your gem easier. You can also use a different console, if you like.
 
 # (If you use this, don't forget to add pry to your Gemfile!)
-# require "pry"
+require "pry"
```

Comment:
@colby why do we have to remove this?? it doesn't hurt anything and makes it easier to be able to bind into things when developing. otherwise we'll just have to add it back in every time you start a new feature branch